gapic/src/main/com/google/gapid/settings.proto

```diff
@@ -142,13 +142,18 @@ message Perfetto {
   }
   CPU cpu = 1;
 
+  message GPUCounters {
+    repeated int32 counter_ids = 1;
+  }
+
   message GPU {
     bool enabled = 1;
     bool slices = 2;
     bool counters = 3;
     int32 counter_rate = 4;
-    repeated int32 counter_ids = 5;
+    reserved 5;
```

Comment:
We may not need this as we distribute the server and client at the same time. You can instead just switch it. You can ask Pascal for details.

Reply:
Actually, the settings are serialized and stored, so it's the only place where it would matter. Although, we currently serialize it to text-format, so we'd be OK re-using tag IDs, it's better practice not to.